<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>client</class>
 <widget class="QDialog" name="client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="server_label">
     <property name="text">
      <string>Server:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="port_label">
     <property name="text">
      <string>Port:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="get_button">
     <property name="text">
      <string>Get Time</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="message_label">
     <property name="text">
      <string>Need connection</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QDateTimeEdit" name="time_edit"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="port_edit"/>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="server_edit"/>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="exit_button">
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
